site/package.json

```diff
@@ -39,7 +39,7 @@
     "@types/jqueryui": "^1.12.16",
     "@types/twig": "^1.12.8",
     "@typescript-eslint/eslint-plugin": "^5.15.0",
-    "@typescript-eslint/parser": "^5.14.0",
+    "@typescript-eslint/parser": "^5.16.0",
```

Comment:
can i work on this issue

Reply:
Hi @suryansh639 This isn't an issue this is a pull request (PR), see GitHub docs for more information here:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests

You can see the list of issues that can worked here, specifically these are the ones tagged 'good first issue':
https://github.com/Submitty/Submitty/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22